### PR TITLE
feature/DEVSU-1697-use-pyproject.toml-with-setup.cfg

### PR DESCRIPTION
**Pull request by RaphaelPletz**
Update setup.py to setup.cfg with pyproject.toml

### CODE REVIEW

**Review by elewis2 — APPROVED** [review]

**Review by elewis2 — APPROVED** [review]

File: setup.cfg [hunk 1]
```diff
@@ -14,3 +14,52 @@ line_length = 100
...
+    mkdocs-material
+    markdown-refdocs
+    flake8
+    black
+    flake8-annotations
+    isort
+    mypy
+
+[options.package_data]
+ipr = ipr/content.spec.json, py.typed
```
Comment (creisle) [inline setup.cfg:61]: are semi-colons and commas interchangeable in toml files? if so can we pick one? or just use the long form lists for everything?

File: setup.cfg [hunk 2]
```diff
@@ -14,3 +14,52 @@ line_length = 100
 multi_line_output = 3
 include_trailing_comma = true
 known_standard_library = requests
+
+
+[metadata]
+name = ipr
+version = 3.1.1
+author_email = graphkb@bcgsc.ca
```
Comment (creisle) [inline setup.cfg:22]: This has been there for a while apparently but could we change this to ipr@bcgsc.ca etc? Its probably a more appropriate author email for this one

File: setup.cfg (hunk 1 above)
Comment (RaphaelPletz) [reply setup.cfg:61]: `are semi-colons and commas interchangeable in toml files?`
No, setup.cfg has list-comma and list-semi types for different fields.

I do believe we can use the long-form lists for everything if you wish so. The docs linked below explain more about the difference, although no form is specified for `package_data`, the list-comma is used in their provided example

https://setuptools.pypa.io/en/latest/userguide/declarative_config.html

Comment (creisle) [reply setup.cfg:61]: ah ok. that's odd but seems like it is a problem on their end

**Review by creisle — APPROVED** [review]

**Pull request merged**